addons/website/static/tests/tours/snippet_countdown.js

```diff
@@ -33,5 +33,14 @@ tour.register('snippet_countdown', {
         trigger: 'h2:contains("Happy Odoo Anniversary")',
         run: () => null,
     },
+    // Next, we change the end action to redirect while the edit message toggle
+    // is still activated. Now, we can check if the countdown wasn't
+    // incorrectly hidden.
+    wTourUtils.changeOption('countdown', 'we-title:contains("at the end") + div we-toggler'),
+    wTourUtils.changeOption('countdown', 'we-button:contains("Redirect")', 'end action'),
+    {
+        trigger: '.s_countdown_canvas_wrapper',
+        run: () => null,
+    },
```

Comment:
@qsm-odoo I double checked and this part of the test does fail when I comment the lines from the same commit.

Reply:
Which precisely is disturbing as, after your first commit, following the flow of your test as a real user leads to **no bug**: https://watch.screencastify.com/v/8oB9zKtMPvb315AiMSbS since your flow was not actually *clicking* on the "hide at the end" button, so you are saying that the test revealed a bug which did not exist.